tests/validation/lib/rke_client.py

```diff
@@ -54,7 +55,9 @@ def build_rke_template(self, template, nodes, **kwargs):
         """
         render_dict = {
             'master_ssh_key_path': self.master_ssh_key_path,
-            'network_plugin': DEFAULT_NETWORK_PLUGIN}
+            'network_plugin': DEFAULT_NETWORK_PLUGIN,
+            'k8s_version': K8S_VERSION
+        }
```

Comment:
Could we also run the other test_install_config tests to see if they are working ? Its a change in the method build_rke_template()  of rke_client.py. Does this change affect other tests ?

Reply:
Does not affect other tests.